src/app/shell/admin-tools/data/users/users.component.ts

```diff
@@ -88,7 +87,8 @@ export class UsersComponent implements OnInit, OnDestroy {
     this.filterFormControl.reset();
     this.childrenParams.searchString = '';
     this.childrenParams.tabTitle = event.tab.textLabel;
-    this.store.dispatch(new GetChildrenForAdmin(this.childrenParams));
+    this.currentPage = PaginationConstants.firstPage;
+    this.getChildren();
```

Comment:
somethign wrong in this method, currently there is double call for getChildren
I suppose this is redundant or should be reconsidered with another approach 
REASONS: 
1) filterFormControl si reset, so it should emit event change. I belive the first call is on 91 row, and then formcontrol reacts on change and does another call. 
2) for the same reason I think this is redundant to reset childrenParams.searchString, because when you reset formControl it reacts in subscription and search string is empty anyway 
![image](https://user-images.githubusercontent.com/27493959/215692964-b11044b0-f35b-4caa-8a46-2fe4efb87a48.png)
![image](https://user-images.githubusercontent.com/27493959/215693049-ab680747-70fc-47fd-86ec-c6348f6b7fdb.png)